prover/src/lib.rs

```diff
@@ -111,8 +110,9 @@ impl Prover for ExecutionProver {
     type Air = ProcessorAir;
     type BaseField = Felt;
     type Trace = ExecutionTrace;
+    type HashFn = Blake3_256;
```

Comment:
The default hash function should be `Blake3_192` for now (here and in other places).

Reply:
Done!